packages/manager/modules/vps/src/dashboard/cancel-resiliation/vps-cancel-resiliation.routing.js

```diff
@@ -0,0 +1,83 @@
+import snakeCase from 'lodash/snakeCase';
+import { BillingService } from '@ovh-ux/manager-models';
+
+export default /* @ngInject */ ($stateProvider) => {
+  $stateProvider.state('vps.detail.dashboard.cancel-resiliation', {
+    url: '/cancel-resiliation',
+    views: {
+      modal: {
+        component: 'billingCancelResiliation',
+      },
+    },
+    layout: 'modal',
+    translations: { value: ['.'], format: 'json' },
```

Comment:
```suggestion

```
we should prefer annotation 
```suggestion
    .run(/* @ngTranslationsInject:json ./translations */)
```

Reply:
This was a result of copy/paste. There are no translation files here and we do not need it. Hence it has been removed. Thanks :)